.gitignore

```diff
@@ -1,2 +1,5 @@
 families.json
-people.json
```

Comment:
Newlines at the end of files stop erroneous diffs like this one. Try to add them in the future